process/graduation_criteria.md

```diff
@@ -0,0 +1,115 @@
+- Encourage public visibility of experiments or other early work that can add value to the mission of AzCU, Cloud Native
+  in Azure and its impact on the larger Cloud Native community.
+- Facilitate alignment with existing projects within Azure as well as the larger community
+- Remove obstacles to adoption and contribution by ensuring projects adhere to Microsoft Open Source Guidelines, code of
+  conduct and IP policy requirements
+  
+### Entry Requirements
+- Required backing of at least one member of the product management
+- Display of the AzCU "Sandbox" badge on website / readme
+- Draft project proposal for review and approval by TOC
+  - A great example is the [CNCF Project Proposal Process](https://github.com/cncf/toc/blob/fc161c13a86ba022277a906a10ff51a568406f7c/process/project_proposals.adoc)
+  - TODO: create our own project proposal process based on the CNCF project proposal process
+- Begin working toward achieving [Core Infrastructure Initiative Best Practices Badge](https://bestpractices.coreinfrastructure.org/)
+  - Must have 
+    - Gated PRs with unit tests
+    - MSFT code of conduct
+    - LCA bot enabled
+
+## Incubating Stage
+An Incubating project is one that has met all of the requirements of the Sandbox stage and is showing accelerating 
+adoption.
+
+### Entry Requirements
+- Minimum of 3 customers using the project in a production or pre-production environment
+- Minimum of 2 maintainers
+- Full due diligence of the TOC
+  - A great example is the [CNCF Due Diligence Guidelines](https://github.com/cncf/toc/blob/fc161c13a86ba022277a906a10ff51a568406f7c/process/due-diligence-guidelines.md)
+  - TODO: create our own DD guidelines based on the CNCF DD guidelines
+- Demonstrate a substantial ongoing flow of commits and merged contributions
+- Have a healthy number of issues and pull requests from external contributors
+- Set a clear, documented versioning scheme
+- Show significant progress toward achieving [Core Infrastructure Initiative Best Practices Badge](https://bestpractices.coreinfrastructure.org/)
+  - Must have 
+    - Unit tests and integration tests covering a majority of the codebase
+    - Linting
+    - Static analysis (where appropriate)
+    - An initial security review
+    - Contributing guide
+    - Issue and PR templates
+    - Public project board with prioritization
+    - Milestones for releases
+- Telemetry for errors and general usage
+  - If not possible, provide reasoning for why
+
+## Graduation Stage
+Graduation Stage project is one that has met all of the requirements of the Incubation stage and has reached a level
+of maturity and polish which will delight customers.
+
+### Entry Requirements
+- Minimum of 30 customers using the project in production
+- Minimum of 4 maintainers (at least one from outside AzCU)
+- Semester project road map published and updated
+- Full due diligence of the TOC with an eye to any miss alignment
```

Comment:
I don't understand this one, what's "with an eye to any miss alignment"?

Reply:
I was intending this to be miss alignment of the team goals. In the CNCF due diligence document, this is defined as alignment to the goals of the CNCF.

Reply:
got it, should be `misalignment` then I think

Reply:
yes, I believe you are correct 🤣